src/App.js

```diff
@@ -1,40 +1,21 @@
-import React from 'react';
-import SimpleCounter from './store/store';
-
+import React,{ useState }  from "react";
 import './App.scss';
-
-// importing the elements Material-UI
-import Button from '@material-ui/core/Button';
-import { makeStyles } from '@material-ui/core';
-// brief entry: import {makeStyles, Button} from '@material-ui/core/';
+import Modal from "./components/modal/Modal.js";
 
 function App() {
-
-  // To apply styles to material elements
-  const useStyles = makeStyles((theme) => ({
-  root: {
-    '& > *': {
-      margin: theme.spacing(1),
-    },
-  }, testButtons: {
-        '&': {
-          margin: 40,
-        }
-      },
-  }));
-
-  const classes = useStyles();
-
+  const [modalActive,setModalActive]=useState(true);
```

Comment:
line should look like this: const [modalActive, setModalActive] = useState(true);